crates/landmass_rerecast/README.md

```diff
@@ -0,0 +1,153 @@
+# landmass_rerecast
+
+A plugin for [Bevy](https://bevyengine.org) to help using
+[bevy_landmass](https://github.com/andriyDev/landmass) and
+[bevy_rerecast_core](https://github.com/janhohenheim/rerecast) together.
+
+## Overview
+
+`landmass_rerecast` allows you to use `bevy_rerecast_core::Navmesh` as the
+navigation mesh in a `landmass` island! To use it:
+
+1. Setup `bevy_landmass` and `bevy_rerecast_core` up as normal.
+2. Add the `LandmassRerecastPlugin` to your app.
+3. Replace uses of `bevy_landmass::Island3dBundle`,
+   `bevy_landmass::NavMeshHandle3d`, and `bevy_landmass::NavMeshHandle`, with
+   `landmass_rerecast::Island3dBundle`, `landmass_rerecast::NavMeshHandle3d`, and
+   `landmass_rerecast::NavMeshHandle`, respectively.
+   1. Note: you can either change the imports, **or** if you are importing the
+      `bevy_landmass` prelude, you can just import these symbols manually.
```

Comment:
Couldn't we expose a `landmass_rerecast::prelude` that the user can just import after the `bevy_landmass::prelude`?

Reply:
Multiple glob imports just makes it ambiguous - order doesn't matter. We could have a prelude that just makes it more obvious which symbols we want users to import. I'm a little nervous to do that since it's fairly common for users to `use whatever::prelude::*;` which would be the wrong pattern!